### ISSUE TITLE
ghz-web linux compiled with 'CGO_ENABLED=0' with go-sqlite3

### ISSUE THREAD

**Issue by btran55** [bojand/ghz#137]
**Command line arguments / config**
List all command line arguments or config properties
`./ghz-web -config web.toml`

```[server]
port = 80   # the port for the http server

[database]  # the database options
type = "sqlite3"
connection = "data/ghz.db"

[log]
level = "info"          # log level
path = "/tmp/ghz.log"   # the path to log file, otherwize stdout is used
```
**Describe the bug**

After running CLI command with default config settings given in docs getting `Binary was compiled with 'CGO_ENABLED=0', go-sqlite3 requires cgo to work. This is a stub`

**To Reproduce**

Download latest binary for linux and try to start ghz-web with default config given in docs. 

**Expected behavior**
it starts

**Environment**
 - OS: Linux
 - ghz: 0.42.0 or 0.41.0

**Comment by Ekliptor** [comment]
any update on this?

**Comment by babayega** [comment]
Any progress on this?

**Comment by bojand** [comment]
Hello, I am a bit constrained with time these days but will try and get to this soonish. As a work around one can build the project with go-sqlite3 support and CGO_ENABLED=1 manually for their target platform.

**Comment by bojand** [comment]
Hello, I believe this should be addressed now in [0.53.0](https://github.com/bojand/ghz/releases/tag/v0.53.0). Please reopen issue if it is still persists.
